src/main/java/org/simulator/io/CSVImporter.java

```diff
@@ -153,7 +153,7 @@ public MultiTable convert(Model model, String pathname) throws IOException {
     if (line != null) {
       header = line.split(",");
       line = reader.readLine();
-      while (line != null) {
+      while ((line != null) && !line.isEmpty()) {
```

Comment:
Not even sure where to start here. The whole read function has many problems.
First thing: this has as a side effect a change in the header instance variable but at the same time returns a string 2d array. This does not make any sense to me.
This complete module should be cleaned up.

1. One function `readDataFromCSV(String pathname)` which returns a HashMap<String, Double[]>. No global side effects, all columns are read and directly accessible via their key, do the parsing to Doubles right here. No need to generate another hashmap on the outside, no need for finding indices of header keys. This should be a public function which can be reused for other purposes.
2. `convert` should have a better name; i.e. `adaptDataToModel` or something similar and should just take a HashMap<String, Double[]> as second argument. Please document what is exactly happening in the method, i.e. how is the data filtered/adapted, ... to fit to the model.